testkit/src/main/scala/org/bitcoins/testkit/wallet/DLCWalletUtil.scala

```diff
@@ -129,8 +130,24 @@ trait DLCWalletUtil {
     val walletA = fundedWalletA.wallet
     val walletB = fundedWalletB.wallet
 
+    val numOutcomes = scala.util.Random.nextInt(20) + 2
```

Comment:
I think it is probably better to just choose some number like `8`, either that or turn some `val`s into `def`s taking `numOutcomes: Int` as an input